docs/manifest.md

```diff
@@ -0,0 +1,156 @@
+  - `ADAPTER` – a site-specific adapter that allows dapplets to work with site specific contexts
+  - `INTERFACE` – a virtual adapter which provides the interface for dapplets for using several site-specific adapters
+  - `LIBRARY` – a dynamic adapter. It provides the work of the all other adapters. It's specified in the extension's settings
+
+    ![set dynamic adapter](/img/manif_03.jpg)
+
+- **title** – a module's name. It's displayed in the extension's dapplets' list, in the Dapplets Store, on the NFT etc.
+- **icon** – a link to the dapplet's icon. It's an optional parameter.
+- **contextIds** – a list of resources where the module is loaded and activated. You should list site domains or a name of the module (adapter, interface), where domains are already listed.
+
+  For example, you can specify certain domains in the adapter's manifest:
+
+  ```json
+  // ./adapter/dapplet.json
+  {
+    "contextIds": [
+      "twitter.com",
+      "www.twitter.com",
+      "mobile.twitter.com",
+      "twitter.com/id",
+      "www.twitter.com/id",
+      "mobile.twitter.com/id"
+    ],
+  }
+  ```
+
+  and specify this adapter in the dapplet's manifest:
+
+  ```json
+  // ./dapplet/dapplet.json
+  {
+    "contextIds": ["twitter-adapter.dapplet-base.eth"],
+  }
+  ```
+
+  So this dapplet will be loaded and activated at the same resources as the adapter.
+
+  There is an opportunity to specify the dynamic context. It means that this context can appear at any time on the page and the module will start working. It may be the tweet or another post:
+
+  ```json
+  {
+    "contextIds": ["twitter.com/1551967807428071431"],
+  }
+  ```
+
+  And the last way is setting the contexts determined by content detectors. Currently they are specified in the Dapplet extension and only one is available – "video" context:
+
+  ```json
+  {
+    "contextIds": ["video"],
+  }
+  ```
+
+  It means that the module works if there are <video\> elements on the page. It is also dynamic context.
+
+- **config** – a dapplet's config. It's an optional field that's used only in dapplets.
+The idea is to add some settings to the dapplet which can be changed in the extension.
+For more information look [here](./#).
+
+- **overlays** – a list of the overlays using with the dapplet. If your dapplet uses the overlay you have to add here its name/ID and the development server:
+
+  ```json
+  {
+    "overlays": {
+      "example-04-overlay": "http://localhost:3000"
+    }
+  }
+  ```
+
+  How to make a dapplet with the overlay you can find here: [Ex04: Overlays](/docs/overlays)
+
+- **dependencies** – adapters which are used in the module. You have to set dependencies for FEATURE and ADAPTER modules.
+In the dapplets you set the site-specific and virtual adapters, in the site-specific adapters – dynamic adapter:
+
+  ```json
+  {
+    "dependencies": {
+      "dynamic-adapter.dapplet-base.eth": "0.6.22"
+    },
+  }
+  ```
```

Comment:
Reminder: the following declaration is also allowed theoretically, but it's not tested.
```json
  {
    "dependencies": {
      "twitter-adapter.dapplet-base.eth": {
         "default": "0.6.22",
         "new": "0.6.22"
      }
    },
  }
  ```

Reply:
Don't think that we need to show it in the docs. Especially it's not tested.